examples/all-clusters-app/esp32/README.md

```diff
@@ -40,13 +40,13 @@ The VSCode devcontainer has these components pre-installed, so you can skip this
 step. To install these components manually, follow these steps:
 
 -   Clone the Espressif ESP-IDF and checkout
-    [v4.3 tag](https://github.com/espressif/esp-idf/releases/v4.3)
+    [v4.4 tag](https://github.com/espressif/esp-idf/releases/v4.4)
 
           $ mkdir ${HOME}/tools
           $ cd ${HOME}/tools
           $ git clone https://github.com/espressif/esp-idf.git
           $ cd esp-idf
-          $ git checkout v4.3
+          $ git checkout release/v4.4
```

Comment:
If we are moving to v4.4 tag (as you've mentioned above), then it should be `git checkout v4.4` ? (at all places)

Reply:
v4.4 tag has not been created yet. We have only the branch.